tests/test_optimizer.py

```diff
@@ -0,0 +1,66 @@
+import multiprocessing as mp
+import time
+
+import pytest
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+import hivemind
+from hivemind.averaging.control import AveragingStage
+from hivemind.optim.experimental.grad_averager import GradientAverager
+
+
+@pytest.mark.forked
+def test_grad_averager():
+    dht1 = hivemind.DHT(start=True)
+    model1 = nn.ParameterDict(dict(w=nn.Parameter(torch.zeros(3))))
```

Comment:
Maybe declare the dict in a regular manner here? It's even a bit more compact:
```suggestion
    model1 = nn.ParameterDict({"w": nn.Parameter(torch.zeros(3))})
```